packages/expect/src/__tests__/__snapshots__/toThrowMatchers.test.js.snap

```diff
@@ -38,126 +40,124 @@ Received has value: <red>\\"a string\\"</>"
+      <dim>at jestExpect (</>packages/expect/src/__tests__/toThrowMatchers-test.js<dim>:24:74)</>"
 `;
 
 exports[`.toThrow() strings did not throw at all 1`] = `
-"<dim>expect(</><red>function</><dim>).toThrow(</><green>string</><dim>)</>
+"<dim>expect(</><red>received</><dim>).</>toThrow<dim>(</><green>expected</><dim>)</>
 
-Expected the function to throw an error matching:
-  <green>\\"apple\\"</>
-But it didn't throw anything."
+Expected error pattern: <green>\\"apple\\"</>
+
+Received function did not throw an exception"
 `;
 
 exports[`.toThrow() strings threw, but message did not match 1`] = `
-"<dim>expect(</><red>function</><dim>).toThrow(</><green>string</><dim>)</>
+"<dim>expect(</><red>received</><dim>).</>toThrow<dim>(</><green>expected</><dim>)</>
+
+Expected error pattern: <green>\\"banana\\"</>
+Received error message: <red>\\"apple\\"</>
 
-Expected the function to throw an error matching:
-  <green>\\"banana\\"</>
-Instead, it threw:
-<red>  Error</>
-<red>      <dim>at jestExpect (</>packages/expect/src/__tests__/toThrowMatchers-test.js<dim>:24:74)</></>"
+      <dim>at jestExpect (</>packages/expect/src/__tests__/toThrowMatchers-test.js<dim>:24:74)</>"
 `;
 
 exports[`.toThrow() strings threw, but should not have 1`] = `
-"<dim>expect(</><red>function</><dim>).not.toThrow(</><green>string</><dim>)</>
+"<dim>expect(</><red>received</><dim>).</>not<dim>.</>toThrow<dim>(</><green>expected</><dim>)</>
+
+Expected error pattern: <green>\\"apple\\"</>
+Received error message: <red>\\"apple\\"</>
 
-Expected the function not to throw an error matching:
-  <green>\\"apple\\"</>
-Instead, it threw:
-<red>  Error</>
-<red>      <dim>at jestExpect (</>packages/expect/src/__tests__/toThrowMatchers-test.js<dim>:24:74)</></>"
+      <dim>at jestExpect (</>packages/expect/src/__tests__/toThrowMatchers-test.js<dim>:24:74)</>"
 `;
 
 exports[`.toThrowError() error class did not throw at all 1`] = `
-"<dim>expect(</><red>function</><dim>).toThrowError(</><green>type</><dim>)</>
+"<dim>expect(</><red>received</><dim>).</>toThrowError<dim>(</><green>expected</><dim>)</>
 
-Expected the function to throw an error of type:
-  <green>\\"Err\\"</>
-But it didn't throw anything."
+Expected error name: <green>\\"Err\\"</>
+
+Received function did not throw an exception"
 `;
 
 exports[`.toThrowError() error class threw, but class did not match 1`] = `
-"<dim>expect(</><red>function</><dim>).toThrowError(</><green>type</><dim>)</>
+"<dim>expect(</><red>received</><dim>).</>toThrowError<dim>(</><green>expected</><dim>)</>
+
+Expected error name: <green>\\"Err2\\"</>
+Received error name: <red>\\"Error\\"</>
 
-Expected the function to throw an error of type:
-  <green>\\"Err2\\"</>
-Instead, it threw:
-<red>  Error</>
-<red>      <dim>at jestExpect (</>packages/expect/src/__tests__/toThrowMatchers-test.js<dim>:24:74)</></>"
+Received error message: <red>\\"apple\\"</>
```

Comment:
oh, love this separation, makes it way easier to trace

Reply:
Proximity for the win :)